style.css

```diff
@@ -0,0 +1,497 @@
+body {
+  margin-left: 0%;
+  margin-right: 0%;
+  margin-top: 0%;
+  margin-bottom: 0%;
+  font-family: 'lato', sans-serif;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+}
+
+.twitter {
+  background-color: black;
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 8%;
+  height: 5vh;
+}
+
+.twitter a {
+  text-decoration: none;
+  color: white;
+  padding-left: 3%;
+  padding-top: 5px;
+}
+
+.navlogo {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding-left: 10%;
+  height: 60px;
+  position: fixed;
+  width: 100%;
+}
+
+.navlogo img {
+  padding-top: 10px;
+}
+
+.seeall a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.partners li {
+  list-style: none;
+  color: rgb(196, 196, 196);
+  font-size: 20px;
+  font-weight: 900;
+}
+
+.navlogo ul {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 16%;
+}
+
+.navlogo ul li {
+  list-style: none;
+  padding-right: 10%;
+  padding-left: 10%;
+}
+
+.navlogo ul li a {
+  text-decoration: none;
+  color: black;
+}
+
+.kamuclass {
+  background-image: linear-gradient(180.45deg, rgba(182, 175, 175, 0) 2.75%, rgba(246, 242, 242, 0.9) 15.94%), url(img-folder/Background1.jpg);
+  background-size: cover;
+  background-position: center;
+  border-top: 1px solid rgba(246, 242, 242, 0.9);
+  padding-left: 12%;
+  padding-bottom: 5%;
+  padding-top: 15%;
+  margin-bottom: -1.5%;
+}
+
+.hello {
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.kamuclass h1 {
+  color: #ec5242;
+  font-size: 40px;
+  font-weight: 900;
+}
+
+.allP {
+  color: #272a31;
+}
+
+.stating-date {
+  font-size: 30px;
+  font-weight: 900;
+  color: #272a31;
+}
+
+.programsclass {
+  background-image: url(img-folder/background2.jpg);
+  color: white;
+  padding-bottom: 15px;
+  padding-top: 10px;
+}
+
+.programsclass h2 {
+  display: flex;
+  justify-content: center;
+}
+
+.linebar {
+  background-color: #ec5242;
+  border-color: #ec5242;
+  height: 1px;
+  width: 40px;
+}
+
+.bloc-programs {
+  display: flex;
+  flex-direction: row;
+  padding-left: 10%;
+  padding-right: 10%;
+  gap: 5px;
+  margin-top: 3%;
+}
+
+.bloc1 {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  background-color: rgba(44, 42, 42, 0.7);
+  padding: 1%;
+  width: 18%;
+}
+
+.bloc1:hover {
+  border: 1px solid #fff;
+}
+
+.bloc1 h3 {
+  color: #ec5242;
+  height: 10%;
+  margin-left: 2%;
+}
```

Comment:
- [ ] As mention by previous reviewer, in the main program section , I can see that there is still a bit of misalignment. As you are already using the `flex box` I suggest you use `flex-basis` property on the` title` (`h3` in `.bloc`). The `flex-basis` CSS property sets the initial main size of a flex item. It sets the size of the content box unless otherwise set with box-sizing. This way even when the sizes of different titles is different the initially designated size to the element will make the rest of the main cards look like they are aligned.

Your design:

<img width="315" alt="Screenshot 2023-03-16 at 9 48 19 PM" src="https://user-images.githubusercontent.com/44798044/225726099-3e4221bc-0ddf-4c7b-a7da-f422a5dbf00b.png">

Behance design :

<img width="469" alt="Screenshot 2023-03-16 at 9 55 16 PM" src="https://user-images.githubusercontent.com/44798044/225726247-252c5a0c-d193-4301-9af4-c4822e560d21.png">